Podcast/FollowerFollowingViewController.swift

```diff
@@ -76,10 +72,14 @@ class FollowerFollowingViewController: ViewController, UITableViewDataSource, UI
         endpointRequest.success = { request in
             guard let follows = request.processedResponseValue as? [User] else { return }
             self.users = follows
-            self.refreshControl.endRefreshing()
+            self.usersTableView.endRefreshing()
             self.usersTableView.stopLoadingAnimation()
             self.usersTableView.reloadSections([0] , with: .automatic)
         }
+        
+        endpointRequest.failure = { _ in
+            self.usersTableView.endRefreshing()
```

Comment:
i got this error in the console, but it kept refreshing?
```
Could not load user subscriptions, request failed
Could not load user favorites, request failed
```

Reply:
These print statements are from ExternalProfileViewController not FollowerFollowingViewController so the endpoint probably hadn't returned yet in your case.
<img width="637" alt="screen shot 2017-11-10 at 4 55 19 pm" src="https://user-images.githubusercontent.com/14966713/32680320-1595f504-c638-11e7-81c9-ac50ddf96284.png">

Reply:
ohhhhh ok!